lib/hash.js

```diff
@@ -1,12 +1,18 @@
 const CACHE_VERSION = require('../package.json').version
-const md5hex = require('md5-hex')
+const crypto = require('crypto')
 const salt = JSON.stringify({
   istanbul: require('istanbul-lib-coverage/package.json').version,
   nyc: CACHE_VERSION
 })
 
 function Hash (code, filename) {
-  return md5hex([code, filename, salt]) + '_' + CACHE_VERSION
```

Comment:
I've looked into this further, the `Hash` function is never actually run.  The only reference to `lib/hash.js` is in `index.js` from `NYC.prototype._createTransform`.  caching-transform does not use `opts.hash` so I think we should remove it.  `opts.salt` is used so I think `lib/hash.js` should `module.exports = {salt}`, delete the Hash function completely.

Reply:
I see what happened, this is unused due to the update from caching-transform@1 to caching-transform@2.  I'm working on a patch to address this part correctly, the Hash function isn't entirely unneeded, it needs to be replaced with other code.